distribution/docs/src/main/resources/content/_managing/_configuring/securing-admin-console.adoc

```diff
@@ -19,8 +19,7 @@ To set access restrictions on the ${admin-console}, consult the organization's s
 . Select the *Configuration* tab.
 . Select the *Web Context Policy Manager*.
 .. A dialogue will pop up that allows you to edit ${branding} access restrictions.
-.. Once you have configured your <<{reference-prefix}org.codice.ddf.security.policy.context.impl.PolicyManager,realms>> in your security infrastructure, you can associate them with ${branding} contexts.
-.. If your infrastructure supports multiple <<{reference-prefix}org.codice.ddf.security.policy.context.impl.PolicyManager,authentication methods>>, they may be specified on a per-context basis.
+.. If your infrastructure supports multiple <<{reference-prefix}org.codice.ddf.security.policy.context.impl.PolicyManager,authentication methods>>, they may be specified.
```

Comment:
I wonder if this sentence would be clearer if we revised it to `If your infrastructure supports multiple authentication methods, you may specify them here.`

I read `they may be specified` as they may already be there, but I think we are trying to say you have the option to specify, if desired.